Lib/site.py

```diff
@@ -289,14 +289,34 @@ def joinuser(*args):
 
     return joinuser("~", ".local")
 
+# Copy of sysconfig.get_platform() but only for MinGW
+def _get_platform():
+    if os.name == 'nt':
+        if 'gcc' in sys.version.lower():
+            if 'ucrt' in sys.version.lower():
+                if 'amd64' in sys.version.lower():
+                    return 'mingw_x86_64_ucrt'
+                return 'mingw_i686_ucrt'
+            if 'clang' in sys.version.lower():
+                if 'amd64' in sys.version.lower():
+                    return 'mingw_x86_64_clang'
+                if 'arm64' in sys.version.lower():
```

Comment:
Wasn't arm32 added to this already in #59?

Reply:
Oops, this branch doesn't have that, I should update this as well as add those new commits to this branch. Thanks for noticing.